app/src/main/java/de/danoeh/antennapod/fragment/SubscriptionFragment.java

```diff
@@ -131,6 +132,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
             case R.id.subscriptions_filter:
                 FeedFilterDialog.showDialog(requireContext());
                 return true;
+            case R.id.subscriptions_sort:
+                FeedSortDialog.showDialog(requireContext());
```

Comment:
This is missing a line `return true;`. Otherwise, it will fall through and execute the next case, too.